test/specs/map/PreviewFeaturesService.spec.js

```diff
@@ -155,6 +160,8 @@ describe('ga_previewfeatures_service', function() {
       });
 
       it('loads then adds new features from their ids', function(done) {
```

Comment:
This test is  a test for bod layer which are non geojson. I suggest you copy this test  and add a new one to test if the features are added when the layer is a geojson.